lib/rainforest/cli/options.rb

```diff
@@ -133,9 +121,21 @@ def foreground?
     end
 
     def validate!
-      if @require_token
+      if !TOKEN_NOT_REQUIRED.include?(command)
         unless token
-          raise ValidationError, "You must pass your API token using: --token TOKEN"
+          raise ValidationError, 'You must pass your API token using: --token TOKEN'
+        end
+      end
+
+      if browsers
+        raise BrowserException, browsers unless (browsers - VALID_BROWSERS).empty?
+      end
+
+      if folder
+        # Allow it to start with `./` but not with just `/`
```

Comment:
I figured this would only be used when at the root of the project that holds your Rainforest tests... but that's a good point.